app/models/listed_taxon.rb

```diff
@@ -631,6 +636,20 @@ def self.species_for_infraspecies(lt)
     lt.save
   end
 
+  def self.get_defaults_for_taxon_place(place,taxon, options = {})
+    options[:limit] ||= 100
+    options[:limit] = 200 if options[:limit] > 200
+    taxon = Taxon.find_by_id(taxon) unless taxon.is_a?(Taxon)
+    return unless taxon
+    place = Place.find_by_id(place) unless place.is_a?(Place)
+    return unless place
+    place_descendant_ids = place.descendants.where('admin_level IN (?)',[0,1,2]).pluck(:id)
```

Comment:
This would be more readable as `.where( "admin_level IN (?)", [Place::COUNTRY_LEVEL, Place::STATE_LEVEL, Place::COUNTY_LEVEL])`